src/core/stack_switching/esbuild.config.mjs

```diff
@@ -4,38 +4,10 @@
  */
 
 import { build } from "esbuild";
-import { readFileSync } from "node:fs";
-import loadWabt from "../../js/node_modules/wabt/index.js";
```

Comment:
Do we still need `wabt` as a build dependency?

Reply:
No, I think it's only a test dependency now.